auto_dev/data/templates/compose/tendermint.yaml

```diff
@@ -12,15 +12,47 @@ services:
     network_mode: ${NETWORK_MODE}
     environment:
       - ID=0
-      - PROXY_APP=tcp://${HOST_NAME}
       - TMHOME=/tendermint/node0
       - CREATE_EMPTY_BLOCKS=true
       - DEV_MODE=0
       - LOG_FILE=node_0.txt
       - LOG_LEVEL=INFO
       - WRITE_TO_LOG=true
-    working_dir: /tendermint
-    command: ["run", "--no-reload", "--host=0.0.0.0", "--port=8080"]
+      - HOST_NAME=${HOST_NAME}
+    working_dir: /app
+    entrypoint: []  # override image's "/usr/bin/flask" entrypoint
+    command:
+      - /bin/sh
+      - -c
+      - |
+        # Try each port using Python socket binding
+        python3 - << 'PYEOF' > /tmp/port.txt
```

Comment:
I cannot accept this
Why not just pass the port you want to the docker compose host port mapping